internal/versions.go

```diff
@@ -81,19 +100,72 @@ func parseVersionManifest(data []byte) (*VersionManifest, error) {
 		return nil, errors.Wrap(err, "failed to parse version manifest")
 	}
 
-	if m.Kind != KIND_VERSION_MANIFEST {
+	if m.Kind != versionManifestKind {
 		return nil, fmt.Errorf("did not understand version kind %s", m.Kind)
 	}
 
 	return &m, nil
 }
 
 // GetVersionManifest parses the version manifest file supplied by the user.
-func (a *App) GetVersionManifest() (*VersionManifest, error) {
-	data, err := ioutil.ReadFile(a.Conf.VersionManifestPath)
+func (a *App) GetVersionManifest(localOnly bool) (*VersionManifest, error) {
+	path := a.Conf.VersionManifestPath
+	if path == "" {
+		return nil, errors.New("missing version manifest path")
+	}
+
+	// Conditionally try ConfigMap from api-server first (bootstrapper only)
+	if !localOnly {
+		logrus.Debug("Querying api-server for runtime mappings ConfigMap")
+		manifest, err := a.versionFromAPIServer()
```

Comment:
should this be versionManifestFromAPIServer?

Reply:
Definitely a good catch! It looks like I fixed this locally but forgot to push :(